x/nexus/keeper/transfer_test.go

```diff
@@ -72,7 +161,22 @@ func TestTransfer(t *testing.T) {
 			}).
 		Then("enqueue transfer should return error",
 			func(t *testing.T) {
-				_, err := k.EnqueueForTransfer(ctx, sender, makeRandAmount(randAsset()), feeRate)
+				_, err := k.EnqueueForTransfer(ctx, sender, makeRandAmount(randAsset()))
+				assert.Error(t, err)
+			},
+		).Run(t, repeated)
+
+	Given("a keeper",
+		func(t *testing.T) {
+			k, ctx = setup(cfg)
+		}).
```

Comment:
Cam't you use .Branch to make this test part of the one above and avoid re-declaring Given?

Reply:
Oh, I duplicated the test and but left it unused. Removing it now.